views/github-create-branch.hbs

```diff
@@ -21,15 +21,47 @@
 <input type="hidden" id="_csrf" name="_csrf" value="{{csrfToken}}" />
 <input type="hidden" id="jiraHost" name="jiraHost" value="{{jiraHost}}" />
 
-{{> navigation hideBackButton=true }}
-
 <section class="gitHubCreateBranch">
   <div class="headerImage">
     <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
   </div>
   <div class="gitHubCreateBranch__title">
-    Create a Branch
+    <div class="gitHubCreateBranch__header">Create a Branch</div>
+    <div class="gitHubCreateBranch__subHeader">
+      Creating a branch for
+      <!--TODO: Need to add icons based on the type of issue(story/task/bug) -->
+      <a href="#">KEY-1</a>
+    </div>
   </div>
+
+  <form id="createBranchForm" class="aui top-label">
+    <div class="field-group">
```

Comment:
Looks like we're missing the GitHub Enterprise Server instance input for server customers.

Reply:
Added the field, but haven't added the condition to display it, which I assume will be done in a separate ticket.